composer.json

```diff
@@ -1,46 +1,88 @@
 {
-    "name": "yiisoft/_____",
-    "type": "library",
-    "description": "_____",
+    "name": "yiisoft/app",
+    "type": "project",
+    "description": "Web Application Yii 3.0",
     "keywords": [
-        "_____"
+        "yii3",
+        "app"
     ],
-    "homepage": "https://www.yiiframework.com/",
-    "license": "BSD-3-Clause",
-    "support": {
-        "issues": "https://github.com/yiisoft/_____/issues?state=open",
-        "forum": "https://www.yiiframework.com/forum/",
-        "wiki": "https://www.yiiframework.com/wiki/",
-        "irc": "irc://irc.freenode.net/yii",
-        "source": "https://github.com/yiisoft/_____"
-    },
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4"
+        "php": ">=7.4",
+        "nyholm/psr7": "@dev",
+        "foxy/foxy": "^1.1@dev",
+        "yiisoft/aliases": "@dev",
+        "yiisoft/assets": "@dev",
+        "yiisoft/cache": "@dev",
+        "yiisoft/cache-file": "@dev",
+        "yiisoft/di": "@dev",
+        "yiisoft/event-dispatcher": "@dev",
+        "yiisoft/log": "@dev",
+        "yiisoft/log-target-file": "@dev",
+        "yiisoft/router": "@dev",
+        "yiisoft/router-fastroute": "@dev",
+        "yiisoft/view": "@dev",
+        "yiisoft/widget": "@dev",
+        "yiisoft/yii-debug": "@dev",
+        "yiisoft/yii-web": "@dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.0",
-        "phan/phan": "^3.0"
+        "codeception/codeception": "@dev",
+        "codeception/module-asserts": "@dev",
+        "codeception/module-phpbrowser": "@dev",
+        "yiisoft/composer-config-plugin": "^1.0@dev"
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\_____\\": "src"
+            "App\\": "src"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Yiisoft\\_____\\Tests\\": "tests"
+            "App\\Tests\\": "tests"
         }
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "3.0.x-dev"
+            "dev-master": "1.0.x-dev"
+        },
+        "config-plugin-output-dir": "runtime/build/config",
```

Comment:
I wonder if it is a good idea to put these into `build/config` instead.

Reply:
remove done.